extensions/amp-story/1.0/amp-story.js

```diff
@@ -2360,6 +2360,32 @@ export class AmpStory extends AMP.BaseElement {
   }
 }
 
+/** @implements {../../../src/render-delaying-services.RenderDelayingService} */
+class AmpStoryRender {
+  /**
+   * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
+   */
+  constructor(ampdoc) {
+    /**
+     * @private {!Element}
+     */
+    this.storyEl_ = ampdoc.getRootNode().querySelector('amp-story[standalone]');
+  }
+
+  /**
+   * Function to return a promise for when it is finished delaying render, and
+   * is ready.  Implemented from RenderDelayingService
+   * @return {!Promise}
+   */
+  whenReady() {
+    if (!this.storyEl_) {
+      return Promise.resolve();
+    }
+
+    return this.storyEl_.signals().whenSignal(CommonSignals.LOAD_END);
```

Comment:
You need to wait for `storyEL_` to upgrade to an AMP element before calling `signals()`

Reply:
This hasn't been addressed.

Reply:
Should be addressed now 😃